### ISSUE TITLE
Implement new hair system

### ISSUE THREAD

**Issue by ashledombos** [makehumancommunity/mpfb2#80]
**Describe the solution you'd like**
I’d find interesting if MPFB could generate some modern hair, like hair cards, particles or even the new geometry nodes hair.

Just an idea, I know there are many other things to think about before even considering this.

**Comment by joepal1976** [comment]
Yes, adding support for geometry nodes hair is on my mental todo list. Of the various aspects of a modeled character, I think hair is one of those which feels the least developed. 

I haven't begun looking at any form of solution yet though, so it's likely quite far into the future unless someone else starts building support for it.

**Comment by joepal1976** [comment]
After some research I can conclude that curve style hair cannot be implemented now. There is no python API implemented in blender for it yet, at least not for adding hair. See https://developer.blender.org/T68981

With no api available, there is nothing mpfb can do about it.

So this is on hold at least until Blender has gotten further with readying the APIs.

**Comment by joepal1976** [comment]
With the work done by Tomas Klecer, this is now viable.

**Comment by joepal1976** [comment]
No longer relevant. The haireditor is available now. Tickets should be formulated as extensions/changes relative to that.
